src/table/body-cell/styles.scss

```diff
@@ -380,19 +373,6 @@ $cell-negative-space-vertical: 2px;
       color: awsui.$color-text-button-normal-active;
     }
 
-    &-form {
```

Comment:
For editing cells we used `overflow: visible` on cell content together with negative margins so that the editor can take more space inside a cell and its focus outline is not cut off. Instead, in the refactored versions we just use different padding for editing cells and there is no longer an issue with the outline.

Reply:
Update: the overflow visible is still needed for editable content that includes dropdowns. I will contribute a new screenshot test to capture that.